docs/data-masking-plugin-functions.md

```diff
@@ -321,6 +306,8 @@ Copy the dictionary files to a directory accessible to MySQL. The [secure-file-p
 
     *Percona Server for MySQL* 8.0.21-12 enabled using the `secure-file-priv` option for gen_dictionary_load().
 
+    [MySQL 8.0.23](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-23.html) deprecated the `gen_blacklist()` function. Use `gen_blocklist()` instead.
```

Comment:
Here and everywhere in this file.
In Percona Server's implementation of the Masking Functions Plugin, the function is still named `gen_blacklist()`.
New terminology ('gen_blocklist()') was only introduced in the Percona Server's Data Masking Component (in this 8.0.34 release)